src/os_posix/os_time.c

```diff
@@ -12,26 +12,26 @@
  * __wt_epoch --
  *	Return the time since the Epoch.
  */
-int
+void
 __wt_epoch(WT_SESSION_IMPL *session, struct timespec *tsp)
 {
 	WT_DECL_RET;
 
 #if defined(HAVE_CLOCK_GETTIME)
 	WT_SYSCALL(clock_gettime(CLOCK_REALTIME, tsp), ret);
 	if (ret == 0)
-		return (0);
-	WT_RET_MSG(session, ret, "clock_gettime");
+		return;
+	WT_PANIC_MSG(session, ret, "clock_gettime");
```

Comment:
@keithbostic How would you feel about changing this panic to a diagnostic assertion that retrieving the time doesn't fail, and then eating any error codes here.

That would mean that any caller of __wt_epoch needs to handle time not moving forward, but that's probably true regardless. I'd feel more comfortable with that change.